.gitlab/ci/on-push.yml

```diff
@@ -160,27 +160,27 @@ validate-content-conf:
       fi
     - section_end "Collect Tests"
 
-    - section_start "Prepare Content Packs for Testing"
+    - section_start "Prepare Content Packs for Testing" --collapsed
```

Comment:
@esharf asked for this part to be not collapsed

Reply:
why not?

Reply:
we would like to see the errors right when we open the job without start digging.
this should be reserved for parts that we are sure that they are stable and they have very long outputs.

Reply:
There are too many log lines in this section, you always need to enter the raw logs to see everything.
Another approach - reduce the number of info logs @ilaner

Reply:
O